File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.js
function solution(num_list) {
    let sum = 0;
    let mul = 1;
    for (let i = 0; i < num_list.length; i++) {
        mul *= num_list[i];
        sum += num_list[i];
    }
    sum = sum * sum;
    if (mul < sum) {
        return 1;
    } 
    return 0;
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.py
def solution(num_list):
    total_sum = 0
    total_mul = 1
    
    for num in num_list:
        total_mul *= num
        total_sum += num
        
    total_sum = total_sum ** 2
    
    if total_mul < total_sum:
        return 1
    return 0